FreeRTOS/FreeRTOS
# Add RISCV CVA6 Demo

Add RISCV cva6 Demo

Description
-----------
Get the main_blinky  and main_full working on CVA6 FPGA
Changes added to: FreeRTOS/FreeRTOS/Demo/RISC-V_cva6
No other changes outside this directory 

Test Steps
-----------
Added details in README.md
Compile clean. Generates FreeRTOSDemo.elf which need to be loaded on CVA6 FPGA board. 

Checklist:
----------

Related Issue
-----------
None

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
